### DISCUSSION TITLE
Session Recording

### DISCUSSION THREAD

**Discussion by midshoreit** [rustdesk/rustdesk#1156] [Q&A]
Greetings. I love RustDesk, but **the one thing keeping me from using it as more than just a backup access app is some session recording. Ideally, it just needs to be a simple screen recorder, preferably locked to the remote computer/device being accessed rather rather than the entire monitor, or all monitors.** It is silly to mention that last part, but AeroAdmin uses ffmpeg to record the screen. Although it does technically record the session, it also records the ENTIRE screen on my side AND all the other monitors on my machine, without settings to adjust that. 

If this question has been posed already, please accept my apologies. I'm not sure where to enter things like this.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/833
